packages/react/src/Box/Box.tsx

```diff
@@ -0,0 +1,210 @@
+import {default as clsx} from 'clsx'
+import React, {PropsWithChildren, useMemo} from 'react'
+import {AnimateProps, useAnimation} from '../animation'
+import type {BaseProps} from '../component-helpers'
+import {BaseSizeScale} from '../constants'
+
+/** * Main Stylesheet (as a CSS Module) */
+import styles from './Box.module.css'
+
+export const BoxSpacingValues = ['condensed', 'normal', 'spacious', ...BaseSizeScale] as const
+type SpacingValues = (typeof BoxSpacingValues)[number]
+
+type ResponsiveSpacingMap = {
+  narrow?: SpacingValues
+  regular?: SpacingValues
+  wide?: SpacingValues
+}
+
+export const BoxBackgroundColors = ['default', 'inset', 'subtle', 'overlay'] as const
+type BackgroundColors = (typeof BoxBackgroundColors)[number]
+
+export const BoxBorderRadiusOptions = ['small', 'medium', 'large', 'full'] as const
+type BorderRadiusOptions = (typeof BoxBorderRadiusOptions)[number]
+
+export const BoxBorderWidthOptions = ['thin', 'thick', 'thicker'] as const
+type BorderWidthOptions = (typeof BoxBorderWidthOptions)[number]
+
+export const BoxBorderColorOptions = ['default', 'muted', 'subtle'] as const
+type BorderColorOptions = (typeof BoxBorderColorOptions)[number]
+
+type BorderStyleOptions = Extract<React.CSSProperties['borderStyle'], 'solid' | 'none'>
+
+type BoxProps = {
```

Comment:
Should we add an `as` property here? I can image box being used to wrap a group of components to add a background color, which can be a `section` or `article` html candidate. Or as a wrapper for a list of `Cards` or `Pillars`, so the Box can be a `ul` item too.

Reply:
`ul` can significantly change the semantic behaviour, and also encourages a dependence on Box that I'm trying to avoid (it's meant for situational one-off styles). If we're trying to suggest that cards and pillars need this functionality, perhaps we need a CardGroup or PillarGroup instead to make this more obvious?

I do think adding section and article landmarks could be useful, but also feel we're bordering into potential accessibility violation waters when we weren't originally intending to. Can we see how Box is being used before committing to this suggestion @josepmartins @danielguillan?

Reply:
I agree 👍  let's see how Box is used and consider this later